scripts/build.sh

```diff
@@ -33,11 +33,10 @@ cp LICENSE $TEMP_DIR/
 
 ls -al $TEMP_DIR
 
-if [[ $(uname -r) != "$ARCH" ]]; then
-    if [[ "$ARCH" == "arm64" ]]; then
-        CONDA_SUBDIR=osx-arm64 conda create -n micromamba micromamba=0.6.5 -c https://conda-web.anaconda.org/conda-forge --yes
-        EXTRA_CONSTRUCTOR_ARGS="$EXTRA_CONSTRUCTOR_ARGS --conda-exe $CONDA_PREFIX/envs/micromamba/bin/micromamba --platform osx-arm64"
-    fi
+if [[ $TARGET_PLATFORM != win-* ]]; then
+  CONDA_SUBDIR=$TARGET_PLATFORM conda create -n micromamba micromamba=0.8.2 -c conda-forge --yes
+  MICROMAMBA_FILE=$(find $CONDA_PREFIX/pkgs -iname micromamba)
```

Comment:
Why use it from `pkgs`?

Reply:
because when I tried on the server, the unmodified micromamba worked fine. There is a prefix replacement going on, and I think it might break micromamba for some reason.

Reply:
Okay. Thanks. Shall we remove the hash from micromamba so that we have stable download links like `https://anaconda.org/conda-forge/micromamba/$VERSION/download/$TARGET_PLATFORM/micromamba-$VERSION-0.tar.bz2`

Reply:
it's a good idea, yeah!